man/systemd.network.xml

```diff
@@ -2257,6 +2257,53 @@
       </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[TrafficControlQueueingDiscipline] Section Options</title>
+    <para>The <literal>[TrafficControlQueueingDiscipline]</literal> section manages the Traffic control. It can be used
+    to configure the kernel packet scheduler and simulate packet delay and loss for UDP or TCP applications,
+    or limit the bandwidth usage of a particular service to simulate internet connections.</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>Parent=</varname></term>
+        <listitem>
+          <para>Specifies Queueing Discipline (qdisc). Takes one of <literal>root</literal> or <literal>clsact</literal>.
+          Defaults to <literal>root</literal>.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetworkDelaySec=</varname></term>
+        <listitem>
+          <para>Specifies the network fixed amount of delay to all packets going out of the interface. Defaults to unset.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetworkDelayJitterSec=</varname></term>
+        <listitem>
+          <para>Specifies the random delay to be added to <varname>NetworkDelaySec=</varname>. Defaults to unset.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetworkDelayLimitPacket=</varname></term>
+        <listitem>
+          <para>Specifies the fifo limit (packets). An unsigned integer ranges 0 to 4294967294. Defaults to 100.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetworkDelayLossQuota=</varname></term>
+        <listitem>
+          <para>Specifies the random packet loss in percent to be added to <varname>NetworkDelaySec=</varname>.
+          A integer ranges 0-100. Defaults to unset.</para>
```

Comment:
s/A/An/
s/ranges/in the range/

And "Defaults to 0." (if you decide to change above as well)

Reply:
No we write unset